tajo-core-tests/src/test/java/org/apache/tajo/engine/query/TestAlterTable.java

```diff
@@ -159,12 +160,15 @@ public final void testAlterTableRepairPartition() throws Exception {
       "36.0,N,1,1\n" +
       "38.0,N,2,2\n" +
       "45.0,R,3,2\n" +
-      "49.0,R,3,3\n";
+      "49.0,R,3,3\n" +
+      "null,,null,null\n" +
+        "null,,null,null\n" +
```

Comment:
If you use the same indentation of previous codes, it will be more better. Other added codes in this class need to be updated.